solr/core/src/test/org/apache/solr/cloud/LeaderTragicEventTest.java

```diff
@@ -53,24 +53,21 @@
 
   private String collection;
 
-  @BeforeClass
-  public static void setupCluster() throws Exception {
-    configureCluster(2)
-        .addConfig("config", TEST_PATH().resolve("configsets").resolve("cloud-minimal").resolve("conf"))
-        .configure();
-  }
-
   @Before
   public void setUp() throws Exception {
     super.setUp();
     collection = getSaferTestName();
+    configureCluster(2)
```

Comment:
I'd add a comment to say that a shared cluster led to sporadic test-failures.  But fine as-is.